package.json

```diff
@@ -7,6 +7,7 @@
     "@antora/cli": "~3.0",
     "@antora/site-generator-default": "~3.0",
     "@antora/site-generator-ms": "git+https://gitlab.com/opendevise/oss/antora-site-generator-ms#as-extension",
+    "@asciidoctor/tabs": "^1.0.0-beta.5",
```

Comment:
It would be best to depend on the exact version whenever using a prerelease. So `1.0.0-beta-5` (no leading caret).